scripts/run_format_check.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+find protostar -name '*.py' | xargs black --check
```

Comment:
I couldn't find a way of doing this without a separate script, `pyproject.toml` seems to be a bit retarded when it comes to running bash instructions.